test/JDBC/input/BABEL-CASE_EXPR-vu-verify.sql

```diff
@@ -3106,3 +3106,2464 @@ SELECT CASE 'abcd' COLLATE ARABIC_CI_AS
    WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS
 END AS RESULT
 GO
+
+--- SYS.SYSNAME tests
+SELECT CASE 1
+   WHEN 1 THEN CAST(N'بطاقة التسجيل - قياسية' AS sys.sysname)
```

Comment:
Please don't add explicit schema name in the datatype.

Reply:
Removed the use of schema name explicitly.